spec/models/picture_spec.rb

```diff
@@ -22,14 +22,14 @@
     it 'is required' do
       subject.extension = nil
       expect(subject.valid?).to be_falsey
-      expect(subject.errors.messages).to eq(:extension => ["can't be blank", "must be a png, jpg, or svg"])
+      expect(subject.errors.messages).to eq(:extension=>["can't be blank", "must be a png, jpg/jpeg, or svg"])
     end
 
     it "accepts only png, jpg, or svg" do
```

Comment:
probably should change this text now (or perhaps change it to ignore the specifics):

```suggestion
    it "accepts only specific file extensions" do
```